doc/source/installation.rst

```diff
@@ -1,140 +1,209 @@
 .. -*- coding: utf-8 -*-
 
 ====================
-Installation
+Installation Guide
 ====================
 
-The latest versions of **MDAnalysis** can be installed using `conda` or `pip`.
-Currently, the conda releases only support serial calculations.
-If you plan to use the parallel OpenMP algorithms, you need to
-install MDAnalysis with pip and have a working OpenMP installation.
+The latest version of **MDAnalysis** can be installed using :ref:`mamba-installation` (recommended), 
+or :ref:`pip-installation`. If you need to install MDAnalysis from source, follow :ref:`source-installation` 
+for stable releases. If you are a contributor or developer working on MDAnalysis, follow the 
+:ref:`development-installation` for editable installation and code modifications.
 
-MDAnalysis has a separate :ref:`test suite <mdanalysistests>` **MDAnalysisTests** that is required to run the test cases and examples.
-The test files change less frequently, take up around 90 MB of space,
-and are not needed for daily use of MDAnalysis. However, they are often used in examples,
-including many in this User Guide. If you are not interested in developing
-MDAnalysis or using the example files, you most likely don't need the tests. If you want to
-run examples in the User Guide, install the tests.
-The tests are distributed separately from the main package.
+Currently, the :ref:`mamba-installation` installs a version of MDAnalysis that does **not** include `OpenMP`_ 
+acceleration due to limitations in precompiled conda packages. If you need `OpenMP`_-enabled features, 
+consider installing via :ref:`pip-installation` and compiling from source with a compatible compiler and 
+OpenMP support.
+
+**MDAnalysisTests** is optional and it is a separate :ref:`test suite <mdanalysistests>` used 
+for verifying MDAnalysis installations and running User Guide examples. It is not required for 
+daily use, but some tutorials rely on it. The package is approximately 90 MB and is not updated frequently. 
+If you plan to use tutorial examples or contribute to MDAnalysis, installing it is recommended.
+
+MDAnalysis requires Python 3.8 or later. Ensure you have an appropriate Python version installed before proceeding.
```

Comment:
Currently 3.10.

Perhaps remove and link to the CHANGELOG or the PyPi page to check the latest supported version, just so that we don't forget updating in the future.